docs/faq/admin.txt

```diff
@@ -109,3 +109,12 @@ Android and iOS.
 
 Depending on feature support, there *may* be minor stylistic differences
 between browsers. These are considered acceptable variations in rendering.
+
+What assistive technologies are supported for using the admin?
+==============================================================
+
+We want the admin to be compatible with wide range of assistive
+technologies, but are aware of many blockers currently. We intend to support
```

Comment:
```suggestion
technologies, but are aware of many current blockers. We intend to support
```
What's the intended meaning of "are aware of many blockers currently"? 🤔 The blockers are current? If so then "many current blockers".

Reply:
👍 "we are currently aware of many blockers" sounds much better